### DISCUSSION TITLE
Accessibility Video

### DISCUSSION THREAD

**Discussion by gabebott** [USU-ITLS-x265-23-summer/class-repo#135] [Show and tell]
In one of my UX design courses a couple semesters ago, I saw this video that I think does a good job of showing what types of things we can include in projects to increase the number of people that it is accessible to. If you have 8 minutes, I would highly recommend watching it!

https://youtu.be/oi3__pUK3C4

**Comment by metasean** [comment]
This is a great video!

Make sure you add it to the class-repo. 😉

**Comment by Skye404** [comment]
Great video that shows a lot having to do with disabilities and how we can make things more accessible to users with unique needs.
